contract-ui/tabs/overview/components/ShareContract.tsx

```diff
@@ -0,0 +1,160 @@
+import {
+  useDashboardEVMChainId,
+  useDashboardNetwork,
+} from "../../../../@3rdweb-sdk/react";
+import {
+  AUDITED_BADGE_HEIGHT,
+  BADGE_HEIGHT,
+  BADGE_WIDTH,
+} from "../../../../constants/badge-size";
+import { useTrack } from "../../../../hooks/analytics/useTrack";
+import {
+  Flex,
+  GridItem,
+  Image,
+  Input,
+  Select,
+  SimpleGrid,
+  useClipboard,
+} from "@chakra-ui/react";
+import { SiTwitter } from "@react-icons/all-files/si/SiTwitter";
+import { useMemo, useState } from "react";
+import { FiCheck, FiCopy } from "react-icons/fi";
+import { Button, Card, Heading, Text } from "tw-components";
+
+type ShareContractProps = {
```

Comment:
```suggestion
interface ShareContractProps = {
```

We use interface for props everywhere else

Reply:
not too hung up on this one, either way is fine